run.go

```diff
@@ -273,7 +272,7 @@ func runAction(clicontext *cli.Context) error {
 		oci.WithMounts([]specs.Mount{
 			{Type: "cgroup", Source: "cgroup", Destination: "/sys/fs/cgroup", Options: []string{"ro", "nosuid", "noexec", "nodev"}},
 		}),
-		oci.WithoutRunMount, // unmount default tmpfs on "/run": https://github.com/containerd/nerdctl/issues/157
+		WithoutRunMount(), // unmount default tmpfs on "/run": https://github.com/containerd/nerdctl/issues/157
```

Comment:
Just `if runtime.GOOS == "linux" { opts = append(opts, oci.WithoutRunMount) }`

Reply:
did it this way becuase `oci.WithoutRunMount` is in a Linux tagged file so it fails when building. :disappointed: